modules/photosynthesis/DESCRIPTION

```diff
@@ -9,18 +9,16 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     workflow management tool that is designed to simplify the management of
     model parameterization, execution, and analysis. The goal of PECAn is to
     streamline the interaction between data and models, and to improve the
-    efficacy of scientific investigation. The PEcAn.photosynthesis package contains
-    functions used in the Hierarchical Bayesian calibration of the Farquhar et al 1980 model.
+    efficacy of scientific investigation. The PEcAn.photosynthesis package
+    contains functions used in the Hierarchical Bayesian calibration of the
+    Farquhar et al 1980 model.
 Depends:
     rjags
 Suggests:
-SystemRequirements: JAGS2.2.0
-OS_type: unix
-License: FreeBSD + file LICENSE
-Copyright: Authors
-LazyLoad: yes
-LazyData: FALSE
-Collate:
-    fitA.R
-    Licor.QC.R
-    plots.R
+    
+SystemRequirements: NA
+OS_type: NA
+License: NA
```

Comment:
Is there a reason that the SystemRequirements, License, Copyright, etc were removed for the photosynthesis module?